packages/@lwc/engine-core/src/patches/detect-non-standard-aria.ts

```diff
@@ -56,32 +55,30 @@ function findVM(elm: Element): VM | undefined {
 }
 
 function checkAndReportViolation(elm: Element, prop: string, isSetter: boolean, setValue: any) {
-    if (!isLightningElement(elm)) {
```

Comment:
I actually couldn't remember, so I wrote a test to confirm the behavior: 892212bb2

I checked, and the reason this works i because we are overriding the `Element.prototype` accessors in `detect-non-standard-aria` _after_ the accessors were applied to `LightningElement.prototype` and `BaseBridgeElement.prototype`. So when you call the accessors directly on the component (internally with `this.aria*` or externally with `elm.aria*`), it never goes through the accessors in this file.

This is kind of confusing, so I added a comment to explain: 8ab13faa9